app/tests/test_jinja_filters.py

```diff
@@ -125,10 +125,7 @@ def test_clean_tags(input_text, expected_output):
         ("series_name", "Series name"),
         ("transferring_body_description", "Transferring body description"),
         ("consignment_reference", "Consignment ref"),
-        # edge cases
-        # empty string should return empty string
         ("", None),
-        # single word, no underscore
         ("singleword", None),
         ("unknown_field", None),
```

Comment:
singleword is probably uneeded, unknown_field field is sufficient, as there is no distinction as to what they are trying to prove: both are showing that a string not in the map results in None. Additionaly I think we should just error on this scenario like I mentioned before, rather than fallback to None, as otherwise if there is an odd scenario where there is an extra field then we would not know about it unless a user raises it with us but it is completely unexpected, so erroring with a 500 makes sense to me!

So OPENSEARCH_FIELD_NAME_MAP[field] would do that instead of OPENSEARCH_FIELD_NAME_MAP.get(field)